tests/ios/frameworks/genrule-inputs/c/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("//rules:framework.bzl", "apple_framework")
+
+genrule(
+   name="lib-src.swift",
+   cmd = """
+cat > $(OUTS) <<-EOF
+public struct C {
+
+    public static func run() {
+       print("runs here") 
```

Comment:
Debugging log that can be removed?

Reply:
For me it was valuable to have sentential to the user know this app was doing anything on startup other than just having a white screen

Reply:
👍